python/pyspark/ml/clustering.py

```diff
@@ -380,11 +395,11 @@ class BisectingKMeans(JavaEstimator, HasFeaturesCol, HasPredictionCol, HasMaxIte
     .. versionadded:: 2.0.0
     """
 
-    k = Param(Params._dummy(), "k", "number of clusters to create",
+    k = Param(Params._dummy(), "k", "The desired number of leaf clusters. Must be > 1.",
```

Comment:
Do we also maybe want to copy the full text here since we are updating it anyways? "The actual number could be smaller if there are no divisible leaf clusters."